docs/eventing/samples/ping-source/service.yaml

```diff
@@ -0,0 +1,11 @@
+# This is a very simple Knative Service that writes the input request to its log.
```

Comment:
Maybe mention that it's an event input request or something?

Reply:
it is used all over, maybe we can migrate to a central doc to talk about this?

Reply:
Maybe time to resurrect https://github.com/knative/docs/issues/170 ?